include/magma/vk/textures/Texture.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file Texture.h
+ * @brief Class contain all information about texture
+ * @version 0.1
+ * @date 2021-02-12
+ */
+#pragma once
+#include <string>
+#include <vulkan/vulkan.h>
+#include <memory>
+
+#include "magma/vk/textures/ImageView.h"
+
+struct TextureInfo
+{
+    VkImageCreateInfo       imageInfo;
+    VkImageViewCreateInfo   viewInfo;
+    VkImageLayout           curLayout;
+    std::string             name;
+};
+
+class Texture {
+public:
+    Texture();
```

Comment:
Do we really need a default constructor? Is it used anywhere?

Reply:
We really need default constructor.

Reply:
Fixed